thesis-de/content-de.tex

```diff
@@ -1770,14 +1770,16 @@ \subsection{Vorbereitung der Evaluierung}
 Softwarelösungen durchgeführt werden können, müssen zuvor im ersten Schritt
 Kriterien mit dem Projektteam in einen Kriterienkatalog aufgenommen werden.
 Anschließend werden diesen Kriterien entsprechende Gewichtungen zugewiesen. Bei
-den Kriterien musste darauf geachtet werden, dass diese während der Beurteilung
-der Anforderungen eine Realisierbarkeit besitzen. Nachdem die einzelnen
-Kriterien zusammen mit dem Projektteam erstellt worden sind, wurden diese im
-Anschluss Herrn Reuter zur abschließenden Überprüfung zur Verfügung gestellt.
-Erst hiernach wurden diese in einem Kriterienkatalog aufgenommen.
+den Kriterien muss während der Beurteilung beachtet werden, dass diese
+innerhalb dem Projekt zur Verfügung stehenden Zeit realisierbar sind. Dabei
+wird das prüfen, testen, sowie realisieren der einzelnen Kriterien beachtet.
```

Comment:
Prüfen, Testen, sowie Realisieren

nichteinfach abschrieben
mit bla bla meinte ich:
Realisieren in der Zeitabschätzung der einzelnen Kriterien beachtet wird
Oder hatl ganz anders geschrieben, denk einfach beim schreiben nach...